README.md

```diff
@@ -34,9 +36,41 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then simply run the helm quick install script (or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options)
+```sh
+./scripts/quick-helm-install.sh 
+```
+The script will add the stackrox helm repository, generate an admin password, install stackrox-central-services, create an init bundle for shared secrets, and finally create one secured cluster.
+If you wish to create more secured clusters you may do so using
+```sh
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+  -f stackrox-init-bundle.yaml \
+  --set clusterName="$CLUSTER_NAME"
+```
+
+#### First use
+
+Follow these steps to get to StackRox UI
+
+1.Setup port forward to central:
+```sh
+kubectl -n stackrox port-forward deploy/central 8000:8443
+```
+
+2.Open <https://localhost:8000> in your browser.
+
+3.If a certificate warning is displayed, you can accept the warnings to install the default self-signed certificate. To configure custom certificates, see the section on "Adding Custom Certificates" in <https://docs.openshift.com/acs/configuration/add-custom-certificates.html>.
+
+4.Log in as `admin` using the password in `stackrox-admin-password.txt`
+
+### Manual Helm Installation
+
+In order to install StackRox via Helm you will first need to follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
```

Comment:
```suggestion
To install StackRox using Helm, you must have `helm` on your system. To install `helm` CLI, follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/).
```

Reply:
I don't see what's wrong with the original sentence, but I don't mind changing it.

Reply:
There is nothing wrong - it just uses some words that are not always clear for the readers from other cultures.

1. "In order to" can be almost always reduced to a "to"
2. "via" is not common and is mostly used in lands where children have learned latin at school. The english `using` is a clear indication that one needs to use a tool to reach the goal.
3. "You will first need to" - there is no need to point to the future here. The user might want to do it right now. Additionally, present tense may also indicate future and this is the form that we mostly see in such manuals.
4. The user does not "need" to follow the manual - they just need to have the Helm binary on their system.

Technical writing is not trivial (even for natives) and you should not feel the pressure to learn it. I am not saying that I mastered it (as it took me about 8 years to learn it and I still make many mistakes) but I thought I would share my thoughts as a part of this review.